cnf-tests/testsuites/e2esuite/dpdk/numa_node_sriov.go

```diff
@@ -280,6 +281,61 @@ var _ = Describe("[sriov] NUMA node alignment", Ordered, func() {
 			g.Expect(actualPod.Status.Message).To(ContainSubstring("Resources cannot be allocated with Topology locality"))
 		}, 30*time.Second, 1*time.Second).Should(Succeed())
 	})
+
+	It("Utilize all available VFs then create a pod with guaranteed CPU and excludeTopology set to True", func() {
+		podDefinition := pods.DefinePod(sriovnamespaces.Test)
+		podDefinition = pods.RedefineWithGuaranteedQoS(podDefinition, "2", "500Mi")
```

Comment:
let's avoid shadowing names. 
I'd name:
barePod
withQos
withNetwork...
and so on. Using the same names is a bit confusing / prone to errors

Reply:
Good point